### PR TITLE
#6226 - Fix console activation for mods with no services property

### LINKED ISSUE #6226: Extension console activation is broken for mods that do not have integrations
Steps:
1. Activate a mod in the extension console, that has had the `services` key removed in the workshop, e.g. chrome-extension://mpjjildhmpddojocokjkgmlkkkfjnepo/options.html#/marketplace/activate/linkedin%2Ftemplate-search

Expected result:
* Activation page is shown for the mod

Actual result:
* Error stack trace shown, can't activate mod

### PR DESCRIPTION
**Pull request by BLoe**
## What does this PR do?

- Fixes #6226 

## Checklist

- [ ] Add tests
- [x] Designate a primary reviewer - @grahamlangford

### CODE REVIEW

File: src/extensionConsole/pages/mods/modals/shareModals/__snapshots__/PublishRecipeModals.test.tsx.snap [hunk 1]
```diff
@@ -15,51 +15,44 @@ exports[`renders cancel publish modal 1`] = `
       class="modal-content"
     >
       <div
-        class="modal-header"
+        class="modal-body"
```
Comment (BLoe) [inline src/extensionConsole/pages/mods/modals/shareModals/__snapshots__/PublishRecipeModals.test.tsx.snap:18]: I'm trying to figure out why this snapshot was affected by my PR 🤔

Comment (BLoe) [reply src/extensionConsole/pages/mods/modals/shareModals/__snapshots__/PublishRecipeModals.test.tsx.snap]: I guess the test timed out or something 🤷‍♂️

File: src/activation/WizardValuesModIntegrationsContextAdapter.tsx [hunk 2]
```diff
@@ -40,7 +40,7 @@ const WizardValuesModIntegrationsContextAdapter: React.FC<Props> = ({
   );
   const integrationDependencies: IntegrationDependency[] =
     modComponentDefinitions.flatMap((component) => {
-      const integrationOutputKeyMap = component.services;
+      const integrationOutputKeyMap = component.services ?? {};
```
Comment (BLoe) [inline src/activation/WizardValuesModIntegrationsContextAdapter.tsx:43]: This is the main bug fix

Comment (twschiller) [reply src/activation/WizardValuesModIntegrationsContextAdapter.tsx:43]: > This is the main bug fix, although I think it's only possible in the workshop to create a mod that is missing the

It's valid to leave of the services property, but the Page Editor, as you mention, will include it even if it's empty

**Pull request merged**